src/scripts/main.js

```diff
@@ -22,7 +22,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  if (Object.keys(element).length !== 0) {
+    const newList = document.createElement('ul');
+
+    for (const key in element) {
+      const newListItem = document.createElement('li');
+
+      data.append(newList);
+      newList.append(newListItem);
+      newListItem.textContent = key;
+
+      if (Object.keys(element[key]).length !== 0) {
+        createTree(element[key], newListItem);
+      }
```

Comment:
эта проверка уже есть в начали функции 
```suggestion
        createTree(element[key], newListItem);
```